doc/dev/developer_guide/basic-workflow.rst

```diff
@@ -221,25 +221,22 @@ Refer to the chapters on testing for details.
 Notes on PR make check test
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The GitHub :ref:`make-check` test is driven by a Jenkins instance.
+The GitHub :ref:`make check<make-check>` test is driven by a Jenkins instance.
 
-Jenkins merges the PR branch into the latest version of the base branch before
-starting the build, so you don't have to rebase the PR to pick up any fixes.
+Jenkins merges your PR branch into the latest version of the base branch before
+starting the build. This means that you don't have to rebase the PR to pick up any fixes.
 
 You can trigger the PR tests at any time by adding a comment to the PR - the
 comment should contain the string "test this please". Since a human subscribed
 to the PR might interpret that as a request for him or her to test the PR,
-it's good to write the request as "Jenkins, test this please".
+we recommend that you address Jenkins directly in the request; for example, write "Jenkins, test this please" or "jenkins test this please".
```

Comment:
I am not sure if "Jenkins, test this please" works, let's only mention "jenkins retest this please"?
Perhaps, we should mention other useful jenkins commands too, in a follow-on PR.

> jenkins retest this please
> jenkins test classic perf
> jenkins test crimson perf
> jenkins test signed
> jenkins test make check
> jenkins test make check arm64
> jenkins test submodules
> jenkins test dashboard
> jenkins test api
> jenkins test docs
> jenkins render docs
> jenkins test ceph-volume all
> jenkins test ceph-volume tox

Reply:
1. Good one, neha. I'll all this list.
2. My impulse to correct the command was correct!